src/Compilers/VisualBasic/Portable/Scanner/KeywordTable.vb

```diff
@@ -304,9 +304,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax
 
         Private Shared Function EnsureHalfWidth(s As String) As String
             Dim result As Char() = Nothing
-
+            Dim ch As Char
```

Comment:
What is the cost of the declaration being redeclared on every iteration of the loop?